chromadb/api/segment.py

```diff
@@ -136,6 +143,13 @@ def create_collection(
         for segment in segments:
             self._sysdb.create_segment(segment)
 
+        self._telemetry_client.capture(
+            ClientCreateCollectionEvent(
+                collection_uuid=str(id),
+                embedding_function=embedding_function.__class__.__name__,
```

Comment:
As discussed offline / in https://github.com/chroma-core/chroma/pull/859, let's sanitize what we're sending to either be one of the chroma built-in EF's, or else just a string which tells us it's a custom function, so we don't leak user's class names.

Reply:
Done. The implementation is ugly because it uses `inspect` and has to import the `embedding_function` modules inline in order to avoid a circular import. Open to other ways to accomplish this

Reply:
We could instead do something like

```python
import inspect

# List of all classes in this module
_classes = [obj for name, obj in inspect.getmembers(sys.modules[__name__], inspect.isclass) if obj.__module__ == __name__]

def get_builtins():
    return _classes
```

in the embedding function module itself, then do 

`from embedding_functions import get_builtins` 

That would encapsulate looking up what embedding functions are built-ins on the emebdding function module itself, and mean we only computed the list once and kept it statically allocated rather than computing it every time we fired the event.